### DISCUSSION TITLE
ncp-uart-nsw_6.7.9_115200.ota issue

### DISCUSSION THREAD

**Discussion by GitHubMe78** [arendst/Tasmota#20279] [Q&A] [answered]
Hi,
I tried in vain to upgrade my Zig_bridge32:
Program Version	13.3.0.1(zig_bridge32)
Build Date & Time	2023-12-18T13:24:42
Core/SDK Version	2_0_14/4.4.6.231122
using this ota file:
https://github.com/arendst/Tasmota/blob/development/tools/fw_SonoffZigbeeBridge_ezsp/ncp-uart-nsw_6.7.9_115200.ota
It seems that the directories used for this ota have changed.
Is there a new version?
Any idea?

**Comment by barbudor** [comment, marked as answer]
By looking a bit in the repo you could have found by yourself this folder

https://github.com/arendst/Tasmota/tree/development/tools/fw_SonoffZigbeeBridge_ezsp

have you got the other link from a Tasmota Doc page that would need an update?

**Reply by GitHubMe78** [reply]
Thanks Barbudor,
I saw those files but get confused by the similar names for different data.
The ones you've indicated are OK
I close the discussion
